scripts/compile_kernel.sh

```diff
@@ -86,6 +95,34 @@ function setup_arm_cross_compiler_toolchain () {
 function setup_linux_kernel_sources () {
   echo "### Check if Raspberry Pi Linux Kernel repository at ${LINUX_KERNEL} is still up to date"
   clone_or_update_repo_for 'https://github.com/raspberrypi/linux.git' $LINUX_KERNEL $LINUX_KERNEL_COMMIT
+  # Parches RT patches
+  if [ ! -z $RT ]; then
+     if [ ! -f $LINUX_KERNEL/patch-4.1.13-rt14.patch ]; then
+<<<<<<< HEAD
```

Comment:
@PabloGN Thanks for the PR. But Travis found a problem with it. Seems you have to resolve this merge conflict.